blog/_posts/2022-01-19-learn-scala.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy, Vincent Derouand, and Valérie Pedroni (Scala Center)
+title: "Learn Scala"
+---
+
+Happy new year 2022! The beginning of the year is a good time to refresh 
+your programming skills. We want to use this opportunity to report on what 
+happened on the MOOCs in 2021, and to share our plans for 2022.
+
+## 2021 Retrospective
+
+In 2021, with the release of Scala 3 we updated most of our MOOCs to Scala 3:
+
+- [Functional Programming Principles in Scala],
+- [Functional Program Design],
+- [Parallel Programming],
+- [Programming Reactive Systems] with Akka.
+
+We also kept the Scala 2 versions of these courses online, but we observed 
+that the majority of the learners took the Scala 3 version (e.g., 97% in the 
+case of Functional Programming Principles in Scala).
+
+The only courses that still use Scala 2 are the courses that depend on Spark,
+which does not yet fully support Scala 3 (see below our plans in this regard).
+
+We also published a new course: [Effective Programming in Scala]. This 
+course teaches programmers everything they need to be ready to work in Scala.
+We are proud to report that this course was one of the [100 most popular 
+courses of the year 2021][rank] (out of the 2900 courses created in 2021 on 
+all the MOOCs platform).
+
+In total, in 2021 more than three thousand people completed a Scala online 
+course! Congratulations to them!
```

Comment:
Thank you to them for trusting us as their teachers and congratulations on completing their courses.

Reply:
Thank you for the suggestion. I rephrased this part in 430b3bf0.